pinot-controller/src/main/java/org/apache/pinot/controller/BaseControllerStarter.java

```diff
@@ -212,6 +213,7 @@ public abstract class BaseControllerStarter implements ServiceStartable {
   protected ExecutorService _rebalancerExecutorService;
   protected TableSizeReader _tableSizeReader;
   protected StorageQuotaChecker _storageQuotaChecker;
+  protected List<UtilizationChecker> _utilizationCheckers;
```

Comment:
I meant something like
```suggestion
  protected final List<UtilizationChecker> _utilizationCheckers = new ArrayList<>();
```